cpp/daal/include/algorithms/algorithm_types.h

```diff
@@ -30,6 +30,8 @@
 #include "data_management/data/data_collection.h"
 #include "services/error_handling.h"
 #include "services/daal_shared_ptr.h"
+#include <stdint.h>
```

Comment:
I doublechecked locally without `#include <stdint.h>` and it works also, should we delete this line?

Reply:
Yes, the lesser includes the better. I've removed this line.